src/Tribe/Rewrite.php

```diff
@@ -456,9 +456,9 @@ protected function get_matcher_to_query_var_map() {
 		 * @param  array  array of the current matchers for query vars.
 		 * @param  self   $rewrite
 		 */
-		$matchers = apply_filters( 'tribe_events_rewrite_matchers_to_query_vars_map', $matchers, $this );
+		$map = apply_filters( 'tribe_events_rewrite_matchers_to_query_vars_map', $map, $this );
 
-		return $matchers;
+		return $map;
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 0c9552d5dce69b8911cf475424e02422</sub>